samples/hello-cmake-oboe/cmake/External_Oboe.cmake

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2017 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set(oboe_version "1.1.1")
+set(oboe_url "https://github.com/google/oboe/archive/${oboe_version}.zip")
+set(oboe_md5 "85b7cefe6046f17f8c7cf62762cbb5c7")
+
+set(oboe_name ${CMAKE_STATIC_LIBRARY_PREFIX}oboe${CMAKE_STATIC_LIBRARY_SUFFIX})
+
+ExternalProject_Add(oboe
+  GIT_REPOSITORY https://github.com/NewProggie/oboe
+  GIT_TAG fix_cmake_install_targets
+  # URL ${oboe_url}
```

Comment:
Note that, in order to make this work, you need to change GIT_REPOSITORY/GIT_TAG with URL and URL_MD5, as the current version on master is missing said install targets